### DISCUSSION TITLE
월드 템플릿이 유니티에서 열리지 않아요

### DISCUSSION THREAD

**Discussion by m030m** [naverz/zepeto-studio-kor#1137] [5. Q&A]
![image](https://user-images.githubusercontent.com/110765650/183285344-330bd191-aa72-48be-af07-0767a97ff782.png)

유니티를 처음 깔고 제페토 월드 템플릿을 import했는데
메뉴얼에 있는 것처럼 템플릿이 자동설정 완료가 뜨지 않고 Assets 목록에 파일이 떠 있습니다.

어떻게 해야할까요?

Unity hub(3.2.0)와 Unity 2020.3.9f1 버전을 사용했습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 처음 import를 완료하셨다면 유니티를 종료후, 다시 열어서 제페토 템플릿으로 프로젝트를 생성하여 보시겠어요?
https://studio.zepeto.me/guides/getting-started-with-templates
